discord/message.py

```diff
@@ -1152,42 +1159,42 @@ async def delete(delay: float):
 
     @overload
     async def edit(
-        self,
-        *,
-        content: Optional[str] = ...,
-        embed: Optional[Embed] = ...,
-        attachments: List[Attachment] = ...,
-        suppress: bool = ...,
-        delete_after: Optional[float] = ...,
-        allowed_mentions: Optional[AllowedMentions] = ...,
-        view: Optional[View] = ...,
+            self,
+            *,
+            content: Optional[str] = ...,
+            embed: Optional[Embed] = ...,
+            attachments: List[Attachment] = ...,
+            suppress: bool = ...,
+            delete_after: Optional[float] = ...,
+            allowed_mentions: Optional[AllowedMentions] = ...,
+            view: Optional[View] = ...,
```

Comment:
Why were these indented? Shouldn't they stay at 4 spaces?

Reply:
pylint sucks

Reply:
lol

Reply:
Actually, they were indented to comply with PEP8. When parameters are formatted like this, they should be further indented to distinguish from the code inside the following block.